### ISSUE TITLE
Incompatibility with LTSC is not documented

### ISSUE THREAD

**Issue by awilfox** [kitamstudios/rust-analyzer.vs#57]
I just tried to install this extension on my newly installed Visual Studio 2022 LTSC, which is 17.10.  It failed to start, and looking in the log file, it was because it was unable to load the language server manifest for version 17.11.

I understand that us LTSC users may not be included in the latest updates, but there was no documentation that it would be incompatible.  I had to dig out a VSIX for the last 2.0 to be able to open the extension.  Again, I am fine with the requirement being there, but it would be really nice to have the requirement documented.  It was frustrating for it to simply not work with no message.

**Comment by parthopdas** [comment]
There would have been an error message on launching VS after installing r-a.vs. Something like but with VS minimum version stated as 17.11

![1729920223872313972991784695350](https://github.com/user-attachments/assets/efcc5b85-36bd-45e7-baea-fd97034999bc)

That is the documentation. If such a dialog didn't appear reactivate this issue.

If the extension and min requirements doesn't suit your needs don't use it.

Else either send a PR or upgrade to min requirements.

I published this extension after building it for myself and add features based my my requirements. This is 1/20 of an engineer working on it not a multinational corp.

**Comment by awilfox** [comment]
> There would have been an error message on launching VS after installing r-a.vs. Something like but with VS minimum version stated as 17.11: [image]
> That is the documentation. If such a dialog didn't appear reactivate this issue.

No such dialogue appeared, which is why I opened the issue in the first place.  The only message that was shown was when I opened the Options dialogue and tried to activate rust-analyzer's options:

![vs2022-ltsc-rust-error](https://github.com/user-attachments/assets/9ab11c16-99c4-4294-9eac-c4440d9b1943)

And the referenced log showed:

![vs2022-ltsc-rust-log](https://github.com/user-attachments/assets/9589a5fe-1800-400e-b6cd-160dcb849c5b)

> If the extension and min requirements doesn't suit your needs don't use it.

I was able to successfully download an older version and it works quite well.  I appreciate this extension and your efforts and, as noted, I simply wanted the issue documented.  This way, any others who are confused and may run into this issue won't think badly about the extension.

> Else either send a PR or upgrade to min requirements.

With the workloads I use, LTSC is better suited, but when LTSC is next updated I will be sure to update to the latest version of the extension.

> I published this extension after building it for myself and add features based on my requirements. This is 1/20 of an engineer working on it without getting paid not a multinational corp.

Your work is very much appreciated.  Thank you!
